lib/screens/weather_screen.dart

```diff
@@ -91,6 +91,7 @@ class _WeatherScreenState extends State<WeatherScreen> {
                 state.weatherModel.weatherDescription.toString(),
                 style: const TextStyle(fontSize: 28, color: Colors.white, fontWeight: FontWeight.bold),
               ),
+
```

Comment:
If u want test crashes, add this code:

TextButton(
    onPressed: () => throw Exception(),
    child: const Text("Throw Test Exception"),
),